new/reclassify_amazon_linux_platform_family.md

```diff
@@ -0,0 +1,31 @@
+---
+RFC: unassigned
+Author: Sean OMeara <sean@chef.io>
+Status: Draft
+Type: <Standards Track, Informational, Process>
+<Replaces: RFCxxx>
```

Comment:
Can you can change Type to 'Standards Track' and remove the ```Replaces``` line when you edit this next to save some work on merge later?